website/content/api-docs/system/mounts-all.mdx

```diff
@@ -1,35 +1,31 @@
 ---
 layout: api
-page_title: /sys/internal/ui/mounts - HTTP API
+page_title: /sys/mounts-all - HTTP API
 description: >-
-  The `/sys/internal/ui/mounts` endpoint is used to manage mount listing
+  The `/sys/mounts-all` endpoint is used to manage mount listing
```

Comment:
Something to be aware of in the documentation is that this endpoint only returns mount paths which the current user has access to. This is very different from most of our endpoints so I think it should be called out. 

Maybe also a callout that this endpoint must be present in the policy for the UI to work as expected.